include/libcyphal/common/cavl/cavl.hpp

```diff
@@ -46,6 +46,10 @@
 
 // NOLINTBEGIN(cppcoreguidelines-pro-bounds-constant-array-index)
 
+namespace libcyphal
+{
+namespace common
+{
```

Comment:
Renamed for...
1. to be consistent with `libcypha/common/cavl/...` folder structure
2. to minimize possible conflict if some project already has it's own CAVL copy (and not necessarily the same version)